components/Network/src/WiFi.cpp

```diff
@@ -112,6 +123,16 @@ esp_err_t WiFi::init()
     ESP_LOGD(TAG, "Init TCP/IP adapter");
     tcpip_adapter_init();
 
+
+    tcpip_adapter_dhcpc_stop(TCPIP_ADAPTER_IF_STA); // Don't run a DHCP client
+    tcpip_adapter_ip_info_t ipInfo;
+    inet_pton(AF_INET, DEVICE_IP, &ipInfo.ip);
+    inet_pton(AF_INET, DEVICE_GW, &ipInfo.gw);
+    inet_pton(AF_INET, DEVICE_NETMASK, &ipInfo.netmask);
+    tcpip_adapter_set_ip_info(TCPIP_ADAPTER_IF_STA, &ipInfo); 
+
+
+
```

Comment:
```suggestion
```

Why is this code here? This breaks the configuration for users. Not all users use the same configuration for STA, using DHCP **shouldn't** be disabled.